.github/dependabot.yml

```diff
@@ -28,6 +28,6 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
-    open-pull-requests-limit: 5
+      day: "wednesday"
```

Comment:
We went for Sunday initially to not queue the CI during workdays. I think once the PR gets merged, it will retrigger anyway.

Reply:
OK, let me back on Sunday.